### ISSUE TITLE
Accessibility: Focus indicator missing

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51791]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

There is no visible focus outline on this element when it receives focus.
![T001_KB_FocusOutlineMissing](https://github.com/user-attachments/assets/3fe852a5-25e5-4e7e-bb69-24f4720ad3c0)

### Component or Page: 

**Page: Checkout Process (Block-Based)**

### Name affected element:

1. Country/Region combobox
2. State combobox"

### Relevant code snippets:

`<select size="1" class="wc-blocks-components-select__select" id="shipping-country" aria-invalid="false" autocomplete="country"><option value="AF" data-alternate-values="[Afghanistan]">Afghanistan</option><option value="AX" data-alternate-values="[Åland Islands]">...Zimbabwe</option></select>`

### Recommended fix:

Modify the CSS in your stylesheet to ensure the element has a 2px solid outline with a 2px outline offset on focus. [Ensure that the comboboxes have keyboard outline focus.]

**Comment by mikejolley** [comment]
This was fixed in https://github.com/woocommerce/woocommerce/pull/51332

**Comment by mikejolley** [comment]
Actually I'm not seeing any focus styles now so something may have regressed. cc @senadir

**Comment by Manussakis** [comment]
I have started looking into this issue.

**Comment by Manussakis** [comment]
This issue has been fixed by the https://github.com/woocommerce/woocommerce/pull/52010 PR and can be closed.

**Dark mode**
![Image](https://github.com/user-attachments/assets/1318fbc7-62c5-4df1-8105-e60d73c32f71)

**Light mode**
![Image](https://github.com/user-attachments/assets/3fa985eb-e4af-41ad-a4aa-5b86f27ed0cd)

cc: @vikrampm1

**Comment by vikrampm1** [comment]
@Manussakis please link the PR.

**Comment by Manussakis** [comment]
> [@Manussakis](https://github.com/Manussakis) please link the PR.

@vikrampm1 Done.
